tardis/apps/publication_workflow/components/PublicationsList.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React, { Fragment, useEffect, useState } from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import PublicationCard from './PublicationCard';
+import { deletePub, retractPub } from './utils/FetchData';
+import PublicationToast from './utils/PublicationToast';
+
+
+const PublicationsList = ({
+  releasedPubsList,
+  retractedPubsList,
+  scheduledPubsList,
+  draftPubsList,
+  onPubUpdate,
+  onResumeDraft,
+  listViewType,
+}) => {
+  const [deleteModalOpen, setDeleteModalOpen] = useState(false);
+  const [retractModalOpen, setRetractModalOpen] = useState(false);
+  const [pubToDelete, setPubTodelete] = useState(-1);
+  const [pubToRetract, setPubToRetract] = useState(-1);
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState('Publication deleted Successfully');
+
+  const handleDelete = () => {
+    deletePub(pubToDelete).then(() => {
+      setDeleteModalOpen(false);
+      setToastMessage('Publication deleted Successfully');
+      setShowToast(true);
+      onPubUpdate('draft');
+    });
+  };
+  const handleRetract = () => {
+    retractPub(pubToRetract).then(() => {
+      setRetractModalOpen(false);
+      setToastMessage('Publication retracted Successfully');
+      setShowToast(true);
+      onPubUpdate('');
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
            onPubUpdate('');
```